examples/models/llama3_2_vision/vision_encoder/model.py

```diff
@@ -4,6 +4,8 @@
 # This source code is licensed under the BSD-style license found in the
 # LICENSE file in the root directory of this source tree.
 
+# pyre-ignore-all-errors
+
```

Comment:
Generally we prefer not to add this. Can they be fixed?

Reply:
My internal pyre service is running into proxy issues that I'm trying to sort out, was thinking could just skip since pyre is getting deprecated very soon

Reply:
Do they pass the oss linter? If so then it's ok.